.env.test

```diff
@@ -15,5 +15,5 @@ OMNIAUTH_TEST_MODE=true
 SIDEKIQ_WEB_UI_USERNAME=sidekiq
 SIDEKIQ_WEB_UI_PASSWORD=sidekiq
 INCLUDE_ACCESSIBILITY_SPECS=false
-MAX_UPLOAD_SIZE_BYTES=10000
+MAX_UPLOAD_SIZE_BYTES=1048576
```

Comment:
Thinking about it, maybe better to have a very different, deliberately arbitrary value here like 5MB so that we can verify that things like the error messaging are actually dynamic based on this config.